parsely_raw_data/event.thrift

```diff
@@ -1,21 +1,18 @@
-"""IDL file as Python module."""
-__license__ = """
-Copyright 2016 Parsely, Inc.
+# Copyright 2016 Parsely, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
This is now a `.thrift` file instead of a string inside a `.py` file.  This just simplifies loading the file, and would potentially let people process events in other languages.